src/packages/LinearProgress/LinearProgress.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './linear-progress.css';
+
+const LinearProgress = ({
+    progress,
+    backgroundColor,
+    progressColor,
+    height,
+    minimal,
+}) => {
+    const outerStyle = { backgroundColor, height };
+    const innerClass = typeof progress === 'undefined' ? 'indeterminate' : 'determinate';
+    const innerStyle = { backgroundColor: progressColor };
+
+    if (progress) {
+        innerStyle.width = `${progress * 100}%`;
+    }
+
+    return (
+        <div
+            className={classNames('d2ui-linear-progress', { minimal: minimal })}
+            style={outerStyle}
+        >
+            <div className={innerClass} style={innerStyle} />
+        </div>
+    );
+};
+
+LinearProgress.propTypes = {
```

Comment:
I don't think we need `progress`, `backgroundColor`, `progressColor`, `height`, or `minimal`. We can just have a default indeterminate linear loading indicator.

If we need a separate look and feel for the loading indicators I would rather enable that by different theme classes than let the developer override it manually.

Reply:
So just to check, I will end up with these props:
- progress (float 0-1)
- padded (bool) (i.e. minimal is default and by adding padding you get the margin top/bottom)

Reply:
I wasn't sure. In my personal experience progress is often a result of doing something like:
```javascript
progress = finishedItems / totalItems;
```
Which then makes it mildly easier to work with a 0-1 range

Reply:
Actually, forget about this. I will change to 0-100 to be more consistent with other libraries (i.e. Material UI, etc) out there.

Reply:
>Which then makes it mildly easier to work with a 0-1 range

- The way it's implemented now that means that we just convert the float to a decimal percentage, so we can attempt to render `33.333333333335%`. We should at least trim it to `33%`.

- Without looking at the docs, I would assume that progress is measured in percentages. I find it more intuitive.